### DISCUSSION TITLE
is there a way to get query variables in errorFormatter?

### DISCUSSION THREAD

**Discussion by capaj** [mercurius-js/mercurius#980] [Q&A]
```
  errorFormatter: (res, ctx) => {
        console.log(ctx.__currentQuery)
    return errResponse
  } 
```
with this I can access the current query, but that does not always paint the whole picture. Is there a way to access query variables?

**Comment by mcollina** [comment]
Likely, but I don't know how.

**Reply by capaj** [reply]
so I guess you would accept a PR to add this, right?
